homeassistant/components/dynalite/config_flow.py

```diff
@@ -25,7 +25,7 @@ async def async_step_import(self, import_info: dict[str, Any]) -> Any:
         host = import_info[CONF_HOST]
         for entry in self._async_current_entries():
             if entry.data[CONF_HOST] == host:
-                if entry.data != import_info:
+                if dict(entry.data) != import_info:
```

Comment:
Why do we need to make a copy?

Reply:
poor mans cast 😃 but you are right, I corrected the type in bridge instead.